<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>360</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.PNG</normaloff>logo.PNG</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Classification</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>561</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Load Data From SQL</string>
        </property>
        <widget class="QProgressBar" name="bar_rule_abc">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>63</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rules ABC</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>63</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rules D</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="bar_rule_d">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="bar_no_rule">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>63</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>No Rule</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btt_load_data">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>110</y>
           <width>87</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Refresh Data</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Logical Group</string>
         </property>
        </widget>
        <widget class="QComboBox" name="select_logical_group">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>110</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>-----</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_UBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_PT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_DC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_BR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_TT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_BT</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>561</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Auto Classification</string>
        </property>
        <widget class="QPushButton" name="btt_load_file">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>87</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load file</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txt_link_to_file">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="btt_start_run">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>87</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="bar_classification">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
        <widget class="QLabel" name="txt_status">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Running: x/y - z parts/s - estimate finish time: t mins</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Train Data</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>561</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Up Load Data to SQL</string>
        </property>
        <widget class="QProgressBar" name="bar_rule_abc_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>441</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="bar_rule_d_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>441</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="bar_no_rule_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>90</y>
           <width>441</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btt_load_MO_table">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>87</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Load MO Table</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>63</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Logical Group</string>
         </property>
        </widget>
        <widget class="QComboBox" name="select_logical_group_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>130</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>----</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_UBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_PT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_DC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_BR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_TT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y_BT</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="btt_load_data_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>87</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Load Product des</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btt_load_data_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>87</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Load PEP file</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>561</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Train Data</string>
        </property>
        <widget class="QPushButton" name="btt_load_data_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>87</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Train Rule ABC</string>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>441</width>
           <height>111</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>History data</string>
          </property>
         </item>
        </widget>
        <widget class="QProgressBar" name="bar_no_rule_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>125</y>
           <width>441</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>561</width>
          <height>171</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Root Folder</string>
        </property>
        <widget class="QPushButton" name="conf_btt_set_motable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>87</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MO Table</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="conf_txt_link_motable">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLineEdit" name="conf_txt_link_termcode">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="conf_btt_set_termcode">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>87</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Term Code</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="conf_txt_link_pepfile">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="conf_btt_set_pepfile">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>87</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PEP File</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="conf_txt_link_hsdatabase">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>110</y>
           <width>441</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="conf_btt_set_hsdatabase">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>87</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>HS database</string>
         </property>
        </widget>
        <widget class="QPushButton" name="conf_btt_set_todefault">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>87</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Set as default</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Reporting</string>
       </attribute>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>200</width>
          <height>144</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
